src/reefState/token/selectedAccountTokenBalances.ts

```diff
@@ -239,6 +248,77 @@ export const loadAccountTokens_sdo = ([
       );
 };
 
+export const loadAllTokens_sdo = ([
+  httpClient,
+  tokenBals,
+  forceReloadj,
+  tokensUpdated,
+]: [AxiosInstance, any, any, any]): Observable<any> => {
+  // TODO check the status of signer - could be loading?
+  const tokensBalMap = new Map(
+    tokenBals.data.map(tb => [tb.data.address, tb.data.balance])
+  );
+  let offset = 0;
+  return queryGql$(httpClient, getAllTokensQuery(offset)).pipe(
+    expand((res: any) => {
+      if (res.data.tokenHolders && res.data.tokenHolders.length > 0) {
+        offset += 320;
```

Comment:
limit should be passed in from here into gql - now it's hardcoded in gql and we need to know the value here